source/Calamari/Kubernetes/Commands/KubernetesApplyRawYamlCommand.cs

```diff
@@ -0,0 +1,116 @@
+            this.commandLineRunner = commandLineRunner;
+            this.fileSystem = fileSystem;
+            this.extractPackage = extractPackage;
+            this.substituteInFiles = substituteInFiles;
+            this.structuredConfigVariablesService = structuredConfigVariablesService;
+            this.statusReportExecutor = statusReportExecutor;
+            Options.Add("package=", "Path to the NuGet package to install.", v => pathToPackage = new PathToPackage(Path.GetFullPath(v)));
+        }
+        public override int Execute(string[] commandLineArguments)
+        {
+            if (!FeatureToggle.GitSourcedYamlManifestsFeatureToggle.IsEnabled(variables))
+                throw new InvalidOperationException(
+                    "Unable to execute Kubernetes Apply Raw Yaml Command because the appropriate feature has not been enabled.");
+
+            Options.Parse(commandLineArguments);
+            var deployment = new RunningDeployment(pathToPackage, variables);
+            var configurationTransformer = ConfigurationTransformer.FromVariables(variables, log);
+            var transformFileLocator = new TransformFileLocator(fileSystem, log);
+            var replacer = new ConfigurationVariablesReplacer(variables, log);
+            var conventions = new List<IConvention>
+            {
+                new DelegateInstallConvention(d =>
+                {
+                    extractPackage.ExtractToStagingDirectory(pathToPackage);
+                    //If using the inline yaml, copy it to the staging directory.
+                    var inlineFile = Path.Combine(Environment.CurrentDirectory, "customresource.yml");
+                    var stagingDirectory = Path.Combine(Environment.CurrentDirectory, "staging");
+                    if (fileSystem.FileExists(inlineFile))
+                    {
+                        fileSystem.CopyFile(inlineFile, Path.Combine(stagingDirectory, "customresource.yml"));
+                    }
+                    d.StagingDirectory = stagingDirectory;
+                    d.CurrentDirectoryProvider = DeploymentWorkingDirectory.StagingDirectory;
+                }),
+                new SubstituteInFilesConvention(new SubstituteInFilesBehaviour(substituteInFiles)),
+                new ConfigurationTransformsConvention(new ConfigurationTransformsBehaviour(fileSystem, variables, configurationTransformer, transformFileLocator, log)),
+                new ConfigurationVariablesConvention(new ConfigurationVariablesBehaviour(fileSystem, variables, replacer, log)),
+                new StructuredConfigurationVariablesConvention(new StructuredConfigurationVariablesBehaviour(structuredConfigVariablesService)),
+                new KubernetesAuthContextConvention(log, commandLineRunner),
+                new GatherAndApplyRawYamlConvention(log, fileSystem, commandLineRunner),
+                new ResourceStatusReportConvention(statusReportExecutor, commandLineRunner)
```

Comment:
Current implementation is that all batches of yaml are applied first and then the resource status is reported.

This can be improved later but right now we are looking to get a walking skeleton done.